compiler/core/src/core/config.re

```diff
@@ -40,22 +40,46 @@ module Workspace = {
       );
   };
 
+  let findPath = (workspacePath: string, fileSuffix: string) => {
+    let searchPath = Path.join([|workspacePath, "**/*?(.)" ++ fileSuffix|]);
+    let searchResults =
+      FileSearch.sync(
+        searchPath,
+        ~options={ignore: ["**/node_modules"]},
```

Comment:
another one we should ignore is `.git`.

What do you think about adding an `ignore` field in lona.json to specify those directory that should be ignored? Also ignoring them for the .component search

Reply:
That sounds good.

Reply:
Updated (though not tested exhaustively). LS doesn't read from lona.json yet, only the compiler.